ctapipe/analysis/tests/test_sensitivity.py

```diff
@@ -101,6 +82,28 @@ def test_SensitivityPointSource():
     np.testing.assert_allclose(eff_area_p.value, gen_area_p.value/2)
 
 
+def test_SensitivityPointSource_sensitivity():
```

Comment:
@cdeil I added a specific test for the sensitivity. Is this what you had in mind?